skt/executable.py

```diff
@@ -782,15 +802,19 @@ def load_config(args):
                             config.get('publisher', 'destination'),
                             config.get('publisher', 'baseurl')]
 
-    if config.has_section('runner') and not cfg.get('runner'):
-        rcfg = {}
-        for (key, val) in config.items('runner'):
-            if key != 'type':
-                rcfg[key] = val
-        cfg['runner'] = [config.get('runner', 'type'), rcfg]
-    elif cfg.get('runner'):
-        cfg['runner'] = [cfg.get('runner')[0],
-                         ast.literal_eval(cfg.get('runner')[1])]
+    cfg['runner'] = {}
+    # Check if the runner type is set on the command line
+    if cfg.get('_name') == 'run' and cfg.get('type'):
+        # Use the runner configuration from the command line
+        cfg['runner']['type'] = cfg.get('type')
+        cfg['runner']['jobtemplate'] = cfg.get('jobtemplate')
+        cfg['runner']['jobowner'] = cfg.get('jobowner')
+
+    # If the config file has a 'runner' section, we should find any values
+    # set there that were not set on the command line.
+    if cfg.get('_name') == 'run' and config.has_section('runner'):
+        for option, value in config.items('runner'):
+            cfg['runner'][option] = value
```

Comment:
I would prefer a more general way of merging configuration file and command-line parameters, one where we don't have to list all the parameters explicitly, but this will have to do. Not required for my ACK.

Reply:
I agree with @spbnick, it'd be good if we can use https://github.com/bw2/ConfigArgParse